chandra_aca/tests/test_aca_image.py

```diff
@@ -115,6 +115,54 @@ def test_slice():
     assert np.all(a2 == im80)
 
 
+def test_ellipsis():
+    from chandra_aca import aca_image
+
+    row0 = 11
+    col0 = 22
+    size = 8
+    inp = np.random.uniform(size=(size, size))
```

Comment:
Having a random input like this in a test means that if it fails then the failure is impossible to reproduce. (As opposed to random-ish testing with hypothesis where the failed case is always captured).

I'm not sure what this randomness is doing which is different from having all unique values like `np.arange(64).reshape(8, 8)`. With that input any failed test will be easier to diagnose since the expected values are obvious to humans.

Reply:
Right. I just didn't worry because the content is not relevant, but I fixed it just now.